packages/mco/components/mco-dashboard/queries.ts

```diff
@@ -28,8 +29,9 @@ export const getLastSyncPerClusterQuery = () =>
 
 // ToDo (epic 4422): Need to update as per updates in the metrics
 export const CAPACITY_QUERIES = {
-  [StorageDashboard.TOTAL_CAPACITY_FILE_BLOCK]: `(label_replace(odf_system_map{target_namespace="openshift-storage"} , "managedBy", "$1", "target_name", "(.*)"))  * on (namespace, managedBy, cluster) group_right(storage_system, target_kind) ${TOTAL_CAPACITY_FILE_BLOCK_METRIC}`,
-  [StorageDashboard.USED_CAPACITY_FILE_BLOCK]: `(label_replace(odf_system_map{target_namespace="openshift-storage"} , "managedBy", "$1", "target_name", "(.*)"))  * on (namespace, managedBy, cluster) group_right(storage_system, target_kind) ${USED_CAPACITY_FILE_BLOCK_METRIC}`,
+  // ToDo: For 4.15, Assuming managedBy is unique for each StorageSystem. Need to add targer_namesapce as an another key.
```

Comment:
I know it is a very odd comment (sorry for that), but I am prefixing all these comment with `// ToDo (epic 4422):` so that I can easily search for them later...
```suggestion
  // ToDo (epic 4422): For 4.15, Assuming "managedBy" is unique for each StorageSystem. Need to add "target_namesapce" as an another key.
```